src/main/scala/viper/gobra/backend/Carbon.scala

```diff
@@ -16,7 +16,7 @@ import scala.concurrent.Future
 
 class Carbon(commandLineArguments: Seq[String]) extends ViperVerifier {
 
-  def verify(programID: String, config: ViperVerifierConfig, reporter:Reporter, program: Program)(executor: GobraExecutionContext): Future[VerificationResult] = {
+  override def verify(programID: String, reporter:Reporter, program: Program)(executor: GobraExecutionContext): Future[VerificationResult] = {
```

Comment:
Nitpick: we should add a space between `reporter:` and `Reporter` to be consistent in our code style.